src/layer/dataset.ts

```diff
@@ -64,7 +64,7 @@ export const DATASET_S2L1C: Dataset = {
   searchIndexUrl: 'https://services.sentinel-hub.com/index/v3/collections/S2L1C/searchIndex',
   findDatesUTCUrl: 'https://services.sentinel-hub.com/index/v3/collections/S2L1C/findAvailableData',
   orbitTimeMinutes: 50.3,
-  minDate: new Date(Date.UTC(2015, 6 - 1, 17, 0, 0, 0)),
+  minDate: new Date(Date.UTC(2015, 6 - 1, 27, 0, 25, 31)), // 2015-06-27T10:25:31Z
```

Comment:
according to the comment, the hour should be 10, not 0.

Reply:
Thanks, fixed!

Reply:
Resolving